priv/repo/migrations/20221023191000_create_products.exs

```diff
@@ -0,0 +1,15 @@
+defmodule Snackium.Repo.Migrations.CreateProducts do
+  use Ecto.Migration
+
+  def change do
+    create table(:products) do
+      add :name, :string
+      add :description, :text
+      add :price, :float
```

Comment:
Don't use floats for anything money related (or anything that requires precision), as they are imprecise.

![image](https://user-images.githubusercontent.com/19409687/197418547-0d25ddbf-6750-41a7-bd6c-123a5f28de27.png)

You should use integer values for that (meaning, use cents as the unit of money). There are even libraries that help operations like that and even can have the currency in account. Checkout https://github.com/elixirmoney/money for example.

Reply:
I will do so.

Thanks for the correction and valuable feedback :pray: